ui/webui/resources/cr_elements/cr_button/cr_button.ts

```diff
@@ -53,6 +53,12 @@ export class CrButtonElement extends PolymerElement {
     if (this.classList.contains('action-button')) {
       kind = 'filled'
     }
+
+    // Avatar buttons should not have a border
+    if (this.classList.contains('avatar') || this.classList.contains('plain')) {
```

Comment:
better because it's more Nala style?

Reply:
and that would be a bigger patch, e.g. in `profile_card`, right?

Reply:
tbh, it'd be about the same size - I could basically set `border: none` in two places, I think.

Once difference is that the `Add` profile button has a different border (dashed instead of solid).